bluesky/bundlers.py

```diff
@@ -777,4 +782,12 @@ async def configure(self, msg):
             obj_set, _ = self._descriptors[name]
             if obj in obj_set:
                 del self._descriptors[name]
-        await self._cache_read_config(obj)
+
+        # nuke the caches, let read re-fill them.  Saves us work
+        # for multiple
+        if obj in self._describe_cache:
+            # assume the caches are coherent!
```

Comment:
The only time I can think of when the caches are not coherent is for the flyer case, and then `obj` will be in the `config_*_cache` but not in the `describe_cache`. I guess we don't need to clear these caches for the flyer case?

Reply:
Also, we didn't document `configure` in the protocols because we thought we were going to deprecate it, does this mean we need to keep it? Or should we think of a different way of invalidating caches? Or is this a very special case that is unlikely to reoccur?

Reply:
@coretl what is the deprecation status for `configure` in the protocols?

the function is there in the `bundlers.py` in the latest main at  the moment